src/box/box.cc

```diff
@@ -6028,7 +6028,7 @@ box_storage_free(void)
 	flightrec_free();
 	audit_log_free();
 	sql_built_in_functions_cache_free();
-	/* fiber_pool_destroy(&tx_fiber_pool); */
+	fiber_pool_destroy(&tx_fiber_pool);
```

Comment:
Better squash this commit with the previous one, which rearranges shutdown sequence.

Reply:
Done.